mne_bids/tests/test_read.py

```diff
@@ -253,6 +267,10 @@ def test_handle_info_reading():
         raw = mne_bids.read_raw_bids(bids_fname, bids_root)
 
 
+@pytest.mark.filterwarnings('ignore:Fiducial point nasion not found'
+                            ':RuntimeWarning:mne')
+@pytest.mark.filterwarnings(r'ignore:The unit for channel\(s\) Status has '
+                            'changed from NA to V.:RuntimeWarning:mne')
```

Comment:
@pytest.mark.filterwarnings('ignore:Fiducial point nasion*:RuntimeWarning:mne')
@pytest.mark.filterwarnings(r'ignore:The unit for chann*V.:RuntimeWarning:mne')